emailserver.js

```diff
@@ -146,7 +182,8 @@ if (process.argv[1] === __filename) {
   server.listen(port);
 } else {
   module.exports = function(cb) {
-    server.listen(0, function(err) {
+    var port = process.env.PORT || 0;
+    server.listen(port, function(err) {
```

Comment:
This makes it possible to test with a background emailserver (although you also have to disable the startup of the one built in to the test). Useful for local development.